base/applications/notepad/dialog.c

```diff
@@ -111,6 +111,33 @@ void UpdateWindowCaption(BOOL clearModifyAlert)
     SetWindowText(Globals.hMainWnd, szCaption);
 }
 
+VOID WaitCursor(BOOL bBegin)
+{
+    static HCURSOR s_hWaitCursor = NULL;
+    static HCURSOR s_hOldCursor = NULL;
+    static INT s_nLock = 0;
+
+    if (bBegin)
+    {
+        if (s_nLock++ == 0)
+        {
+            if (s_hWaitCursor == NULL)
+                s_hWaitCursor = LoadCursor(NULL, IDC_WAIT);
+            s_hOldCursor = SetCursor(s_hWaitCursor);
```

Comment:
I will try to check tomorrow whether all this gymnastics is actually necessary or not. (Because the wait cursor is a standard system one.)